kombu/utils/url.py

```diff
@@ -120,3 +118,20 @@ def maybe_sanitize_url(url, mask='**'):
     if isinstance(url, string_t) and '://' in url:
         return sanitize_url(url, mask)
     return url
+
+
+def parse_ssl_cert_reqs(query_value):
+    # type: (str) -> Any
+    """Given the query parameter for ssl_cert_reqs, return the SSL constant or None."""
+    if ssl_available:
+        query_value_to_constant = {
+            'CERT_REQUIRED': ssl.CERT_REQUIRED,
+            'CERT_OPTIONAL': ssl.CERT_OPTIONAL,
+            'CERT_NONE': ssl.CERT_NONE,
+            'required': ssl.CERT_REQUIRED,
+            'optional': ssl.CERT_OPTIONAL,
+            'none': ssl.CERT_NONE,
```

Comment:
doesn't this break earlier behavior if someone had specified ssl.CERT_REQUIRED?

Seems like you should have also add:

```
            'CERT_REQUIRED': ssl.CERT_REQUIRED,
            ssl.CERT_REQUIRED: ssl.CERT_REQUIRED,
            'CERT_OPTIONAL': ssl.CERT_OPTIONAL,
           ssl.CERT_OPTIONAL: ssl.CERT_OPTIONAL,...
            'CERT_NONE': ssl.CERT_NONE,
           ssl.CERT_NONE: ssl.CERT_NONE,
            'required': ssl.CERT_REQUIRED,
            'optional': ssl.CERT_OPTIONAL,
            'none': ssl.CERT_NONE,
```

Reply:
never mind - misread above